packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,260 @@
+You must [collect and store these credentials](#collect-and-save-customer-credentials) for each customer to allow your integration to work with your customer's Okta org.
+
+### API service integration Client Credentials flow
+
+At a high-level, the OAuth 2.0 Client Credentials flow for an API service integration has the following steps:
+
+<div class="full">
+
+![Flowchart that displays the back and forth between the resource owner, authorization server, and resource server for API service integration's Client Credentials flow](/img/oin/api-service-creds-flow.svg)
+
+</div>
+
+<!-- Source for image. Generated using http://www.plantuml.com/plantuml/uml/
+skinparam monochrome true
+participant "Your app (customer tenant)" as client
+participant "Authorization Server (customer tenant Okta org)" as okta
+participant "Resource Server (customer tenant Okta APIs)" as app
+
+autonumber "<b>#."
+client -> okta: Access token request to /token
+okta -> client: Access token response
+client -> app: Request with access token
+app -> client: Response
+-->
+
+1. Your customer's service app instance makes an access token request to their Okta Authorization Server using their client credentials.
+
+   Your customer needs to install and authorize your integration in their Okta org so that Okta can accept the access token request. See [Set up an API service integration](https://help.okta.com/okta_help.htm?type=oie&id=csh-api-service-integrations) for customer tenant Admin Console instructions. After your integration is installed, the customer obtains their client credentials and passes them to your team so that they can configure the customer's app instance. See [Collect and save customer credentials](#collect-and-save-customer-credentials). The customer's app instance can now make an access token request to Okta. See [Request for access token](#request-an-access-token).
+
+2. If the credentials in the token request are accurate, Okta responds with an access token.
+
+   The resource and action scopes allowed for the access token is configured by you when you [register your API service integration](#register-your-api-service-integration) through the OIN Manager. In addition, the token request must contain allowed scopes for your API service integration. See [Scope selection](#scope-selection).
+
+3. Your customer's service app instance uses the access token to make authorized requests to their Okta org APIs (the resource server). See [Make Okta API requests](#make-okta-api-requests).
+
+4. The customer's Okta org (resource server) validates the token before responding to the API request.
+
+## Build your API service integration
+
+This section provides you with a guideline of the required components, artifacts, and requests to build your API service integration using basic cURL commands. You can use an OAuth 2.0 SDK in your preferred language to implement the OAuth 2.0 Client Credentials flow.
```

Comment:
I found that the Okta Management SDKs don't support client secrets, but do you mean a generic, non-Okta OAuth library?

Reply:
Yes, I mean generic OAuth 2.0 SDK/Libraries (like the one in the blog example: https://docs.spring.io/spring-security/site/docs/5.4.5/reference/html5/#oauth2Client-client-registration). I'm up for suggestions to make this clearer. Maybe if I use "library" instead of "SDK"?

Reply:
IMHO, library gives a clearer picture of an external utility rather than an OKTA SDK.